src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, item) => {
```

Comment:
The use of `reduce` here is appropriate as it accumulates the key-value pairs into an object. However, ensure that the variable `acc` is not named as `result` or similar, as per the checklist item 2.